src/main/reactjs/src/styles/components/clerkTranslator/_authorisation-details.scss

```diff
@@ -7,5 +7,9 @@
     & &--centered {
       text-align: center;
     }
+
+    & &__delete-outline-icon {
```

Comment:
Voisi nimetä myös `delete-icon` jos importissa nimetään näin

Reply:
Tämä poistui kokonaan.